pyaedt/q3d.py

```diff
@@ -554,3 +555,116 @@ def __init__(
             close_on_exit,
             student_version,
         )
+
+    def create_rectangle(self, position, dimension_list, name="", matname=""):
+        """
+        Create a rectangle.
+
+        Parameters
+        ----------
+        position : list
+            List of  [x, y] coordinates for the starting point of rectangle
+        dimension_list : list
+            List of [width, height] dimensions.
+        name : str, optional
+            Name of the rectangle. The default is ``None``, in which case
+            the default name is assigned.
+        matname : str, optional
+            Name of the material. The default is ``None``, in which case
+            the default material is assigned.
+        Returns
+        -------
+        pyaedt.modeler.Object3d.Object3d
+            3D object.
+
+        """
+        pos = self.modeler.Position(*position, 0)
+        return self.modeler.primitives.create_rectangle(pos, dimension_list=dimension_list, name=name, matname=matname)
+
+    def assign_single_signal_line(self, target_objects, name="", solve_option="SolveInside", thickness=None, unit="um"):
+        """
+        Assign conductor type to sheets.
+
+        Parameters
+        ----------
+        name : str
+            Name of the conductor.
+        target_objects : list
+            List of Object3D.
+        solve_option : str
+            Choose among "SolveInside", "SolveOnBoundary" or Automatic. Default is SolveInside.
+        thickness : float
+            Conductor thickness. When not specified, the default conductor thickness is obtained by dividing the
+            conductor's area by its perimeter (A/p). If multiple conductors are selected, the average conductor
+            thickness is used for the default value.
+        unit : str
+            Thickness unit.
+        Returns
+        -------
+        None.
```

Comment:
I don't think we include a "Returns" section is nothing is returned. Someone else can confirm. I'll review all the other docstrings added after the merge.